src/main/java/frc/robot/pilot/Pilot.java

```diff
@@ -56,8 +57,8 @@ public void setupTeleopButtons() {
         controller.a().and(noBumpers()).whileTrue(RobotCommands.laserCanFeed());
         controller.a().and(leftBumperOnly()).whileTrue(LauncherCommands.stopMotors());
 
-        controller.b().and(noBumpers()).whileTrue(RobotCommands.feedToAmp());
-        controller.b().and(leftBumperOnly()).onTrue(RobotCommands.subwooferReady());
+        controller.b().and(noBumpers()).whileTrue(PivotCommands.home());
+        controller.b().and(leftBumperOnly()).onTrue(PivotCommands.subwoofer());
```

Comment:
when done testing, revert pilot or exclude pilot changes from your merge-ready commits as we want to keep the controllers as predictable as possible for driver/operator